GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -1107,8 +1119,6 @@ private void ClearUpdatePlaceholderErrors()
 
         private void UpdatePlaceholders()
         {
-            this.ClearUpdatePlaceholderErrors();

```

Comment:
Because we are updating placeholders multiple times within one command we need to keep this error list around until the command ends so that all the update placeholder errors will be output for the user.  If we clear here, one index changed hook notification would clear it and it would not be output.  We could output with the hook being called but then our output could be intermixed with git's output which would be messy.